src/components/TextField/TextField.tsx

```diff
@@ -132,21 +132,29 @@ class TextField extends React.PureComponent<CombinedProps, State> {
 
     this.state = {
       height: null,
-      focus: false,
+      focus: props.focused || false,
       id: props.id || getUniqueID(),
     };
   }
 
-  componentDidUpdate({focused}: CombinedProps) {
-    if (
-      this.input &&
-      focused !== this.props.focused &&
-      this.props.focused === true
-    ) {
+  componentDidMount() {
+    if (this.input && this.props.focused) {
```

Comment:
The ref should always be available in `componentDidMount`? We should be able to omit that check. The JS styleguide also prefers early returns so we can probably change this to
```tsx
if (!this.props.focused) return;
this.input.focus();
```

Reply:
Sounds good!